package.json

```diff
@@ -71,7 +71,7 @@
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-standard": "^3.0.1",
     "fs-extra": "^5.0.0",
-    "gatsby": "2.18.6",
+    "gatsby": "^2.19.12",
```

Comment:
Change to `    "gatsby": "2.19.12",` and rerun `npm i` so we don't end up on a broken Gatsby version like has happened in the past :)